app/dashboard/templates/bounty/details.html

```diff
@@ -442,6 +442,8 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
   <script src="{% static "v2/js/tokens.js" %}"></script>
   <script src="{% static "v2/js/amounts.js" %}"></script>
   <script src="{% static "v2/js/clipboard.js" %}"></script>>
+  <script src="{% static "v2/js/lib/popper.min.js" %}"></script>
```

Comment:
this is already loaded in footer_scripts.js

Reply:
updated